lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -3556,11 +3559,54 @@ def test_exam_no_redirect(self):
         self.section2.is_time_limited = True
         self.store.update_item(self.section2, self.user.id)
 
-        lms_url, mfe_url = self._get_urls()  # lint-amnesty, pylint: disable=unused-variable
+        lms_url, __, __ = self._get_urls()
 
         assert self.client.get(lms_url).status_code == 200
 
 
+@ddt.ddt
```

Comment:
You can drop this line. You only need it if you use `@ddt.data` (data-driven test data) on any of the class's test functions, which you don't.